tests/e2e/specs/wcpay/shopper/shopper-checkout-purchase-with-upe-methods.spec.js

```diff
@@ -41,88 +40,30 @@ describe( 'Enabled UPE with deferred intent creation', () => {
 
 	describe( 'Enabled UPE with deferred intent creation', () => {
 		it( 'should successfully place order with Giropay', async () => {
-			await shopperWCP.goToShopWithCurrency( 'EUR' );
 			await setupProductCheckout(
 				config.get( 'addresses.customer.billing' )
 			);
-			await selectGiropayOnCheckout( page );
+			await selectOnCheckout( 'giropay', page );
 			await shopper.placeOrder();
-			await completeGiropayPayment( page, 'success' );
+			await completeRedirectedPayment( page, 'success' );
 			await page.waitForNavigation( {
 				waitUntil: 'networkidle0',
 			} );
 			await expect( page ).toMatch( 'Order received' );
 		} );
 
-		it( 'should successfully place order with the default card', async () => {
+		it( 'should successfully place order with Bancontact', async () => {
 			await setupProductCheckout(
 				config.get( 'addresses.customer.billing' )
 			);
-			await fillCardDetails( page, card );
+			await selectOnCheckout( 'bancontact', page );
 			await shopper.placeOrder();
-			await expect( page ).toMatch( 'Order received' );
-		} );
-
-		it( 'should process a payment with authentication for the 3DS card', async () => {
-			await setupProductCheckout(
-				config.get( 'addresses.customer.billing' )
-			);
-			await fillCardDetails( page, config.get( 'cards.3ds' ) );
-			await shopper.placeOrder();
-			await confirmCardAuthentication( page, '3DS' );
+			await completeRedirectedPayment( page, 'success' );
 			await page.waitForNavigation( {
 				waitUntil: 'networkidle0',
 			} );
 			await expect( page ).toMatch( 'Order received' );
 		} );
-
-		it( 'should successfully save the card', async () => {
-			await setupProductCheckout(
-				config.get( 'addresses.customer.billing' )
-			);
-			await fillCardDetails( page, card );
-			await shopperWCP.toggleSavePaymentMethod();
-			await shopper.placeOrder();
-			await expect( page ).toMatch( 'Order received' );
-
-			// validate that the payment method has been added to the customer.
-			await shopperWCP.goToPaymentMethods();
-			await expect( page ).toMatch( card.label );
-			await expect( page ).toMatch(
-				`${ card.expires.month }/${ card.expires.year }`
-			);
-		} );
-
-		it( 'should process a payment with the saved card', async () => {
-			await setupProductCheckout(
-				config.get( 'addresses.customer.billing' )
-			);
-			await shopper.goToCheckout();
-			await uiUnblocked();
-			await shopperWCP.selectSavedPaymentMethod(
-				`${ card.label } (expires ${ card.expires.month }/${ card.expires.year })`
-			);
-			await shopper.placeOrder();
-			await expect( page ).toMatch( 'Order received' );
-		} );
-
-		it( 'should delete the card', async () => {
```

Comment:
Duplicates of

* `shopper-checkout-save-card-and-purchase.spec.js::should process a payment with the saved card`
* `shopper-checkout-save-card-and-purchase.spec.js::should delete the card`
thus removing them.